src/app/pages/server-details/server-details.component.ts

```diff
@@ -1208,7 +1243,92 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
 
       return undefined;
     }
+  }
+
+  public generateStaticWebChart(chartConf: any) {
```

Comment:
this is not a static web chart anymore .. maybe rename to something more general?